### ISSUE TITLE
a11y: Fix keyboard navigation and focus display issues on the account and api keys screen

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#12777]
a11y Issue: 90

Fix keyboard navigation and focus issues on the account and api screen:

**Table Actions Menu**

![image](https://github.com/user-attachments/assets/5963ba81-e123-41b6-b82d-bc4db5e1faf1)

Note: this should be fixed in the sortable table so that it addresses action menu accessibility everywhere it is used.

**Comment by aalves08** [comment]
Will keep this issue open for devs to test but it should be fixed by https://github.com/rancher/dashboard/pull/13019. @rak-phillip going to assign this to you so that you can double-check this. Thanks

**Comment by a-arias** [comment]
**TESTS**
 Rancher : v2.11-d5a1481d04357fd04de2492ee1e7d4576ccca44c-head
 UI: master 3665e77

 Fix verified ✅ 

Automation added on:
https://github.com/rancher/qa-tasks/issues/1703
